editor/animation_track_editor.cpp

```diff
@@ -6377,8 +6396,8 @@ void AnimationTrackEditor::goto_prev_step(bool p_from_mouse_event) {
 		return;
 	}
 	float anim_step = animation->get_step();
-	if (anim_step == 0.0) {
-		anim_step = 1.0;
+	if (anim_step == 0) {
+		anim_step = 1;
```

Comment:
Not too important, but I think this change was unnecessary, we prefer to be explicit with float literals and use the `.0` decimal for whole values (same below).